app/classifier.py

```diff
@@ -51,59 +49,66 @@ def _load_instance(self, path_to_model: str):
 
     def _load_vocab(self, path_to_vocab: str):
         with open(path_to_vocab, 'r') as f:
-            self.vocab = f.readlines()
+            self.vocab = [line.rstrip() for line in f.readlines() if line.rstrip() != '']
```

Comment:
optional nit: you can also do [line.rstrip() for line in f.readlines() if line.rstrip()] since '' evaluates to False. https://stackoverflow.com/a/9573259